feat(frontend): team mentors pages

urlDev opened the pull request:
Created team and mentors pages. Created a reusable card component to use in both team and mentors pages. Fixed footer position on different pages. Changed favicon to the new logo. Changed header image with an inspirational image from unsplash. Changed padding for the card under tl;dr. Changed links from a to next/link. For now, keeping the placeholders down until changing the introduction with putting the placeholder under it maybe.

Added active class to page links. Aligned social media icons in cards, the Twitter icon was staying off to the top.

evedes commented on README.md:
```diff
@@ -19,7 +19,7 @@ If you want to tackle an issue:
     1. create a fork from iamfreecodecamp repo
     2. create a feature branch
-    3. create a PR with detailed and well documented descriptions about what your doing.
+    3. create a PR to our `develop`branch with detailed and well documented descriptions about what your doing.
```
There's a typo, lacking a space between `develop` and branch

evedes commented on components/card.js:
```diff
@@ -0,0 +1,37 @@
+import { FaTwitter, FaLinkedin } from 'react-icons/fa';
+
+export default function Card({ team }) {
```
Maybe team isn't the best variable name to use here? 🤔  Each Card renders a user? person? individual?

evedes commented on components/header.js:
```diff
@@ -1,25 +1,38 @@
+import { useRouter } from 'next/router';
 import { FaTwitter } from 'react-icons/fa';
 
+// const links = [
```
We can remove this commented out code.

evedes commented on pages/mentors.js:
```diff
@@ -0,0 +1,31 @@
+    name: 'Eduardo Vedes',
+    location: 'PT',
+    image: '/assets/team/eduardo.jpg',
+    title: 'Software Developer',
```
I'd prefer Software Engineer here. It has a more wide touch :)

evedes commented on pages/mentors.js:
```diff
@@ -0,0 +1,31 @@
+    location: 'PT',
+    image: '/assets/team/eduardo.jpg',
+    title: 'Software Developer',
+    description: 'Self-taught developer focusing on React, Vue and Node'
```
Selt-taught developer focused on the JavaScript ecosystem. freeCodeCamp Advocate, Tech Writer.

evedes commented on pages/team.js:
```diff
@@ -0,0 +1,35 @@
+  return (
+    <Layout>
+      <div className="md:grid grid-cols-2 gap-4 mt-8">
+        {teamMembers.map((team) => (
```
Instead of team maybe member? 🤔

evedes commented on pages/team.js:
```diff
@@ -0,0 +1,35 @@
+import Card from '../components/card';
+import Layout from '../components/layout';
+
+const teamMembers = [
```
This object is kind of repeated on mentors and team pages. Maybe create a variable in libs or utils folder and export it from there with all the key/values needed for both situations? 🤔

evedes reviewed (commented):
Code looks great! Just left you some minor comments here and there. Besides that let's try to not commit package-lock.json when we don't add something new to package.json. Also, we're using yarn so you should do `yarn` instead of `npm install` :)

urlDev commented:
Fixed background color, card responsive sizes. Added custom height for the header image. Created a utils folder for team/mentors variables. Created a card-list component to style the list of cards from there instead of pages. Changed descriptions.
